_data/team.yml

```diff
@@ -145,15 +151,16 @@
   image: /assets/img/team_agnes.jpeg
 - name: Anne Ferger
   sortkey: Ferger
+  orcid: 0000-0002-1382-2658
   email: anne.ferger@uni-paderborn.de
   affiliation:
     - nfdi
   image: /assets/img/team_anne.jpg
 - name: Daniel Jettka
   sortkey: Jettka
+  orcid: 0000-0002-2375-2227
   email: daniel.jettka@uni-paderborn.de
   affiliation:
-    - nfdi
```

Comment:
usually we keep the former affiliations. Of course if @daniel-jettka wants this removed please remove it